### DISCUSSION TITLE
PnpCore SDK with a legacy project in .Net Framework 4.6.1+

### DISCUSSION THREAD

**Discussion by mahesh139** [pnp/pnpcore#1510] [Q&A]
I searched for documentation and sample to use  PnpCore SDK with legacy .Net framework application but could not find any information on this topic.

SDK documentation says "We currently target .NET Standard 2.0, [.NET 6.0] [.NET 7.0] and [.NET 8.0].  Use the .NET 6.0 (LTS), .NET 7.0 or .NET 8.0 (LTS) builds if you're using a modern .NET version, **use .NET Standard 2.0 for backwards compatibility with .NET Framework 4.6.1+.**"

Is it possible to use this SDK in a project with  .Net Framework  4.6.1+?  

if its possible, any guidance and direction about using it would be very helpful.

Regards,
Mahesh

**Comment by jansenbe** [comment]
@mahesh139 : from https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0. IN theory it should work, but the .NET compatibility matrix mentions it's best upgrade to .NET Framework 4.7.2 or higher.

![image](https://github.com/user-attachments/assets/be1362bc-0be0-40c6-9c14-4ba19bb99a5d)
